src/hash.cr

```diff
@@ -430,6 +432,12 @@ class Hash(K, V)
     nil
   end
 
+  # Returns an `Array` populated with the results of each iteration in the given block.
+  #
+  # ```crystal
```

Comment:
According to [documenting code](http://crystal-lang.org/docs/conventions/documenting_code.html) there is no need for the language to be specified in the comment blocks as it assumes the code is already Crystal.